docs/lab/08/index.md

```diff
@@ -0,0 +1,435 @@
+---
+sidebar_position: 8
+description: Wi-Fi
+slug: /lab/08
+---
+
+# 08 - Wi-Fi
+
+This lab will teach you the basics of networking and Wi-Fi, and how to use Wi-Fi with the Raspberry Pi Pico W, in Embassy.
+
+## Resources
+
+**Andrew Tanenbaum**, *Computer networks (5th edition)*
+  - Chapter 1 - *Introduction*
+    - Subchapter 1.4.2 - *The TCP/IP Reference Model*
+
+## Networking basics
+
+A network is a set of devices that are linked together in order to share resources. Devices can talk to one another using different communication protocols, usually Ethernet (wired) or Wi-Fi (wireless).
+
+Devices that communicate in a network have an associated IP (Internet Protocol) address. There are two types of IP addresses: IPv4 (stored on 32 bits) and IPv6 (newer, stored on 64 bits). We will be discussing only the IPv4 in this lab, since it is most commonly used in local networks.
```

Comment:
This is for TCP/IP networks only. Of course, these are the most commonly used. In any network devices have addresses, in IP networks these are IP addresses.